src/longopt/psbarb_inc.h

```diff
@@ -25,42 +25,19 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'I', "",
-	          "",                  "",
+	GMT_C_CPT_KW,
+	{ 0, 'D', "offset",            "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'G', "fill",              "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'I', "intensity|illumination", "", "", "", "", GMT_TP_STANDARD },
+	{ 0, 'N', "noclip",
+	          "c,r",               "clip_norepeat,repeat|noclip_repeat",
 	          "",                  "",
 		  GMT_TP_STANDARD },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'Q', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'T', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'W', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'Z', "",
-	          "",                  "",
+	{ 0, 'Q', "barbs",
 	          "",                  "",
+	          "a,g,p,j,s,w,z",     "angle,fill,pen,justify,longspeed,width,uvdata",
```

Comment:
`longspeed`? Shouldn't be `windspeed`?

Reply:
That's the speed value which gets drawn as a long barb. _longbarbwindspeed_ seemed a bit long to me.

Reply:
Sorry, I must be missing something, both  `longspeed` and `windspeed` have exactly the same length and **+s** is meant to set the `the wind speed which corresponds to a long barb`